<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitiationQtClass</class>
 <widget class="QMainWindow" name="InitiationQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InitiationQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>status de connexion : deconnecte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessageEditLine">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Message recu =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="far">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>°F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hr">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InitiationQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cel</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>EnvoiCel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>far</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>EnvoiFar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hr</sender>
   <signal>clicked()</signal>
   <receiver>InitiationQtClass</receiver>
   <slot>EnvoiHyg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>920</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDisplayMessageButtonClicked()</slot>
  <slot>onConnectButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onServerNewConnection()</slot>
  <slot>EnvoiCel()</slot>
  <slot>EnvoiFar()</slot>
  <slot>EnvoiHyg()</slot>
 </slots>
</ui>
